webapp/src/main/webapp/iaf/gui/views/SendJmsMessage.html

```diff
@@ -59,6 +59,18 @@ <h4>Send JMS Message</h4>
 								<span class="form-control" style="border: none;"><input icheck type="checkbox" name="lookupDestination" ng-model="form.lookupDestination" /></span>
 							</div>
 						</div>
+						<div class="row form-group">
+							<label class="col-sm-7 col-md-3 control-label label-height-30">Message Property</label>
+							<div class="col-sm-7 col-md-3">
+								<input class="form-control" name="propertyKey" type="text" ng-model="form.propertyKey"/>
+							</div>
+							<div class="col-sm-7 col-md-1">
+								<label class="control-label label-height-30 pull-right">Value</label>
```

Comment:
Why is there a `pull-right` over here?
Please ensure that when using col-*-n the total 'per line' value never crosses 12.

Reply:
it puts too many space between the Value text and the text box. Line 86 has the same expression.

Reply:
Not quite, like 86 has `hidden-xs` thus it only does pull-right when the screen is larger then 992px.
Right now it always does the pull right, which might look ok when you're on a wide screen, but not when it's smaller

Reply:
punt